api/app/controllers/webhook.controller.ts

```diff
@@ -154,3 +165,171 @@ export async function create(req: Request, res: Response) {
     }
   }
 }
+
+async function processCommit(
+  commit: any,
+  pushEvent: any,
+  type: "added" | "modified",
+  branch: string
+) {
+  const changedFiles = [...commit[type]]; // get the files that were added or modified in the commit so we don't have to maintain two separate functions, this is also easily extendable when we want to take care of deleted commits.
+  for (const file of changedFiles) {
+    const rawUrl = `https://raw.githubusercontent.com/${pushEvent.repository.full_name}/${branch}/${file}`;
+    const response: any = await axios.get(rawUrl);
+    const mdContent = response.data;
+    const jsonContent: BaseParsedMdContent =
+      parseMdToJSON<BaseParsedMdContent>(mdContent);
+    const transcript_by = jsonContent.transcript_by.toLowerCase();
+
+    function isTranscriptValid(transcript_by: string): boolean {
+      return (
+        transcript_by.includes("tstbtc") &&
```

Comment:
we should probably convert `transcript_by` to lowercase before doing the includes check